### PR TITLE
Drop python2 module support on 15sp1-ltss

### PR DESCRIPTION
**Pull request by skliu**
quick PR

- Needles: none
- Verification run: https://openqa.suse.de/tests/11200130

### CODE REVIEW

**Comment by jknphy** [comment]
Please fix it in this way, relaxing condition for example:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17147/files
`if (is_sle('=15-sp1') && $output =~ /Python 2 Module.*2021-07-3(0|1)/) {`

**Comment by skliu** [comment]
> Please fix it in this way, relaxing condition for example: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17147/files `if (is_sle('=15-sp1') && $output =~ /Python 2 Module.*2021-07-3(0|1)/) {`

Done.

File: tests/console/zypper_lifecycle.pm [hunk 1]
```diff
@@ -37,7 +37,8 @@ our $date_re = qr/[0-9]{4}-[0-9]{2}-[0-9]{2}/;
 
 sub lifecycle_output_check {
     my $output = shift;
-    if (is_sle('=15-sp1') && $output =~ /Python 2 Module.*2021-07-31/) {
+
```
Comment (jknphy) [inline tests/console/zypper_lifecycle.pm:40]: please remove extra line.
We new code we can separate logically parts, but for existing code shared I don't recommend it, just change the minimum number of lines and characters :)

Comment (skliu) [reply tests/console/zypper_lifecycle.pm]: Done.

**Comment by dzedro** [comment]
I don't know to what date it can change, maybe expect any digit ?

**Comment by skliu** [comment]
> I don't know to what date it can change, maybe expect any digit ?

In fact, I think we can just match `Python 2 Module` and ignore all the date digit since we drop supporting python2 on sle15sp1.

**Comment by dzedro** [comment]
> > I don't know to what date it can change, maybe expect any digit ?
> 
> In fact, I think we can just match `Python 2 Module` and ignore all the date digit since we drop supporting python2 on sle15sp1.

Yes, I was thinking to ignore this for LTSS, at some point this will happen on each product which get LTSS, IMO.

**Comment by jknphy** [comment]
LGTM, we can relax further the condition, the VR seems to take too much time.

**Review by dzedro — APPROVED** [review]

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**